wordpress.org/public_html/wp-content/plugins/wporg-polyglots-posts/wporg-polyglots-posts.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace WordPressdotorg\Plugin\Polyglots_Posts;
+
+/**
+ * Plugin Name: WordPress.org Polyglots Posts.
+ * Description: Filters the posts by their tags, excluding the posts with the
+ * 'editor-requests', 'request', 'clpte', 'locale-requests' tags in the
+ * https://make.wordpress.org/polyglots/?exclude=requests URL.
+ * License: GPLv2 or later
+ */
+
+/**
+ * The ids of the tags to be excluded.
+ * 'editor-requests', 'request', 'clpte', 'locale-requests' id tags.
+ *
+ * @var array
+ */
+const TAG_IDS = array( 1453, 307, 6039, 1999 );
+
+/**
+ * The URL where this plugin will work.
+ *
+ * @var string
+ */
+const URL = 'https://make.wordpress.org/polyglots/?exclude=requests';
+
+/**
+ * Excludes the tags from the main query.
+ *
+ * @param $query
+ *
+ * @return void
+ */
+function pre_get_posts( $query ) {
+	global $wp;
+
+	if ( URL != home_url( add_query_arg( $_GET, $wp->request ) ) ) {
```

Comment:
Using absolute URLs here doesn't seem worthwhile, just check the GET variable is set or not IMHO.

```suggestion
	if ( empty( $_GET['exclude'] ) || 'requests' !== $_GET['exclude'] ) {
```